README.md

```diff
@@ -367,53 +381,81 @@ const counterView = ({ count }: CounterViewInput) => div([
 Because it will be easier going forward `counterView` takes an object
 with a `count` property.
 
-### Output from HTML components
+### Output from components
 
 The above covers the _input_ to the counter view. We now need to get
-_output_ from it. All components in Turbine can produce output.
-Components are represented by a generic type `Component<A>`. The `A`
-represents the output of the component.
+_output_ from it.
+
+Remember that we mentioned that a Turbine component is a huge
+description about what the component will behave and look like. Part
+of that description also explains what output will come from the
+component. Components are represented by a generic type
+`Component<A>`. The `A` represents the output of the component. I.e.
+`Component<A>` means: "I'm a component that will give you `A` as
+output".
+
+To get a feel for what "output" means it may be helpful to mention a
+few examples.
+
+* A button will output a stream of click events. So the type of a
+  button may be `Component<Stream<Event>>`
```

Comment:
Also here maybe better a more explicit `output` in the form `{clickStream: 'click'}`,
which would result in the `{clickStream: Event}` output type and then `Component<{clickStream: Stream<Event>}>` type?

It not clear to me how to get ``Component<Stream<Event>>`, unless you apply the `map`, but that wouldn't be the button itself.